# inputs
in_rgb_topic: /image_rgb
in_nir_topic: /image_nir

# outputs
out_exgr: /piros_vegsec/exgr
out_ndvi: /piros_vegsec/ndvi
out_mask: /piros_vegsec/mask

# selection of index
mask_index: exgr # "exgr" or "ndvi" are the options

# hyperparam
gb_k_size: 9        # gaussian blur kernel size
morph_k_size: 5     # opening and closing structural element size
gsd: 0.33           # ground sample distance [mm/px]
min_area_blob: 50   # min area to be considered vegetation [mm2]
plot_small_obj: False   # this will make smaller blobs gray, to debug
